tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -829,7 +829,7 @@ tensorflow::Status ConvertAfterShapes(ConversionParams& params) {
     // The allocator is used to build the engine. The build and the built engine
     // will be destroyed after we get the serialized engine string, so it's fine
     // to use unique_ptr here.
-    std::unique_ptr<nvinfer1::IGpuAllocator> alloc;
+    std::unique_ptr<TRTDeviceAllocator> alloc;
```

Comment:
Can we use TRTBaseAllocator here?

Reply:
We could. The following code is only using TRTDeviceAllocator tho.
Will update to TRTBaseAllocator